main.js

```diff
@@ -140,22 +166,19 @@ function addPaletteToSavedPalettes(palette) {
     var newMiniContainer = document.createElement('div');
     newMiniContainer.classList.add('mini-container', 'hover');
     savedPalettesSection.appendChild(newMiniContainer);
-    for (i = 0; i < palette.length; i++) {
-        newMiniContainer.innerHTML += `<div class="mini-box", style="background-color: #${palette[i]}"></div>`;
+    for (i = 0; i < palette.hexes.length; i++) {
+        newMiniContainer.innerHTML += `<div class="mini-box", style="background-color: #${palette.hexes[i]}"></div>`;
     }
     newMiniContainer.innerHTML += `<img class="delete-button" src='./assets/delete.png'></img>`;
```

Comment:
We should add in some HTML to see the palette saved name.

Reply:
do you mean other than when a named saved palette is recalled?
<img width="746" alt="Screenshot 2023-04-17 at 10 32 08 AM" src="https://user-images.githubusercontent.com/32660020/232517961-cd36ac4c-444f-4936-9a55-4b745c9ad279.png">

Reply:
hahaha no I was thinking a name here:
![image](https://user-images.githubusercontent.com/123283073/232617011-d22a0b8c-bbc6-4036-98bf-49c1b534184e.png)